.devops/kubernetes/helmCharts/templates/service.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: hub-spid-login-ms
+spec:
+  type: ClusterIP
```

Comment:
the type can be LoadBalancer?

https://docs.microsoft.com/it-it/azure/aks/kubernetes-walkthrough#run-the-application

Reply:
in SelfCare project we have a single Ingress instance, whose externalIp is assigned from terraform project and used inside the apim configuration to forward the request to it.
Maybe we could transform it into a parameter and let this information to be configured as an environment variable (as done for the PORT)

Reply:
ClusterIp is correct, no changes needed